src/fmu/sumo/sim2sumo/main.py

```diff
@@ -54,9 +55,24 @@ def parse_args():
     return args
 
 
+# fmu-dataio needs these when creating metadata
+REQUIRED_ENV_VARS = ["_ERT_EXPERIMENT_ID", "_ERT_RUNPATH"]
+
+
 def main():
     """Main function to be called"""
     logger = logging.getLogger(__file__ + ".main")
+
+    missing = 0
+    for envVar in REQUIRED_ENV_VARS:
+        if environ.get(envVar) is None:
+            print(f"{envVar} is not set.")
+            missing += 1
+
+    if missing > 0:
+        print("Missing required environment variables. Stopping.")
```

Comment:
This could be somewhat meaningless if provided with no context... I.e. this is user-facing, so should probably be a bit more elaborated.

E.g. something like "Required ERT environment variables were not found. This can be because sim2sumo was called outside the ERT context. Stopping."

Reply:
Updated to print "Required environment variable <ENV VAR> is not set." for each missing env var.
Then printing "Required ERT environment variables not found. This can happen if sim2sumo was called outside the ERT context. Stopping."